src/index.html

```diff
@@ -15,8 +15,111 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header
+      id="top"
+      class="header"
+    >
+      <div class="headerImgContainer">
+        <a href="#top">
+          <img
+            class="headerImg"
+            src="./images/logo.png"
+            alt="Moyo header logo"
+          />
+        </a>
+      </div>
+
+      <nav class="navigation">
+        <ul class="list">
+          <li class="listItem">
+            <a
+              href="#1"
+              class="is-active link"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="listItem">
+            <a
+              href="#2"
+              class="link a"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="listItem">
+            <a
+              href="#3"
+              class="link b"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="listItem">
+            <a
+              href="#4"
+              data-qa="hover"
+              class="link c"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="listItem">
+            <a
+              href="#5"
+              class="link d"
```

Comment:
Nie mam pojęcia o co chodzi z tymi klasami 'a', 'b' , 'c' i tak dalej. Wiec to do usunięcia

Reply:
Testowałem coś i zapomniałem to usunąć. Będzie poprawione.